tpl/bootstrap.php

```diff
@@ -41,38 +41,38 @@
 endif;
 ?>
 		<noscript><link type="text/css" rel="stylesheet" href="css/noscript.css" /></noscript>
-		<script type="text/javascript" data-cfasync="false" src="js/jquery-3.4.1.js" integrity="sha512-bnIvzh6FU75ZKxp0GXLH9bewza/OIw6dLVh9ICg0gogclmYGguQJWl8U30WpbsGTqbIiAwxTsbe76DErLq5EDQ==" crossorigin="anonymous"></script>
+		<script type="text/javascript" data-cfasync="false" src="js/jquery-3.4.1.js" integrity="sha512-9anGruNHwVXk3XlsUXFrdEe8Iq5EdB/Otrz+4C+VWtQGPThhPyQRCKPh8+H1QPyu2NmEi5oPuCPACVXPmhnvrQ==" crossorigin="anonymous"></script>
```

Comment:
Why did all these SRI hashes change, for files that didn't get updated in the commit? Maybe regenerate them once more?

Reply:
I checked, but I do get no change detected when doing the phpunit on  f05e5c2, so those SRI hashes are correct and the application works there.

Are you maybe using this on a non-UNIX system that changes the line feeds when you git clone? That would explain the symptom you have... Maybe try it on Janitor, where phpunit etc. are integrated into the WebIDE and you can get a life preview?

Reply:
Ha, yeah! That's it! Well, does not matter, we can regenerate them later.